package.json

```diff
@@ -43,7 +43,7 @@
     "request": "^2.87.0",
     "request-capture-har": "^1.2.2",
     "rimraf": "^2.5.0",
-    "semver": "atlassian-forks/node-semver.git#refs/heads/alwasIncludePrerelease",
```

Comment:
Why was this changed sorry?

Reply:
See: https://github.com/atlassian-forks/yarn/pull/6#issuecomment-895010465